fitnessllm_dataplatform/stream/strava/services/silver_etl_interface.py

```
@@ -3,19 +3,20 @@
 
 from fitnessllm_dataplatform.entities.enums import FitnessLLMDataSource
 from fitnessllm_dataplatform.services.etl_interface import ETLInterface
-from fitnessllm_dataplatform.utils.logging_utils import logger
 from fitnessllm_dataplatform.utils.query_utils import get_delete_query, get_insert_query
 
 
 class SilverStravaETLInterface(ETLInterface):
     """Silver ETL interface for Strava data."""
 
-    def __init__(self, athlete_id: str):
+    def __init__(self, uid: str, athlete_id: str):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)